src/styles/blocks/nav.scss

```diff
@@ -0,0 +1,47 @@
+.nav {
+  &__list {
+    display: flex;
+    justify-content: flex-end;
+    align-items: center;
+    padding: 0;
+    margin: 0;
+  }
+
+  &__item {
+    list-style-type: none;
+    margin-left: 20px;
+  }
+
+  &__link {
+    position: relative;
+    display: flex;
+    align-items: center;
+    text-decoration: none;
+    color: $text-main;
+    height: 60px;
+  }
+
+  &__link:hover {
+    color: rgba(0, 172, 220, 1);
+  }
+
+  &__item:first-child {
+    margin-left: 0;
+  }
```

Comment:
Check the 9th point from the [checklist](https://github.com/mate-academy/layout_catalog/blob/master/checklist.md)